Sources/GRPCHTTP2Core/Server/Connection/ServerConnectionManagementHandler+StateMachine.swift

```diff
@@ -44,12 +44,12 @@ extension ServerConnectionManagementHandler {
       minPingReceiveIntervalWithoutCalls: TimeAmount,
       goAwayPingData: HTTP2PingData = HTTP2PingData(withInteger: .random(in: .min ... .max))
     ) {
-      let keepAlive = KeepAlive(
+      let keepalive = KeepAlive(
```

Comment:
Do we want to rename `KeepAlive` to `Keepalive`? I really dislike it but I wonder if it may be slightly more consistent :D

Reply:
Good catch, yes we should rename it. It might look weird, but it is correct: https://en.wikipedia.org/wiki/Keepalive